woocommerce/woocommerce-ios
# Account deletion: move the CTA from empty stores cell to ellipsis menu action sheet

Closes: #8178 


### Description


To make the Close Account CTA less obvious on the empty stores screen, the CTA is now moved to the ellipsis menu action sheet.

### Testing instructions


- Log out if needed
- Log in to an account without any sites or sign up for a new account --> the empty stores screen should be shown, and there should no Close Account CTA in the 
- Tap `Close account` --> an alert should be shown to close the account


### Screenshots


before | after
-- | --
![Simulator Screen Shot - iPhone 14 Pro - 2022-12-01 at 11 32 55](https://user-images.githubusercontent.com/1945542/204959827-81d65c10-cbe9-4af4-9147-9b5eb24fffb4.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2022-12-01 at 11 31 12](https://user-images.githubusercontent.com/1945542/204959818-280cc412-d7d9-4852-97d3-d56caaf26ad3.png)
![Simulator Screen Shot - iPhone 14 Pro - 2022-12-01 at 11 32 58](https://user-images.githubusercontent.com/1945542/204960276-a06af41a-2ad4-4764-9976-ee3e6b74f34e.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2022-12-01 at 11 37 13](https://user-images.githubusercontent.com/1945542/204960278-81261a6c-3749-4df5-b46a-bd76816790e9.png)


---
